### DISCUSSION TITLE
Create new organization falls into a permanent loop.

### DISCUSSION THREAD

**Discussion by matfeinikolaev** [supabase/supabase#2699] [Questions] [answered]
After filling the name in the "Create new organization" form, it redirects to https://app.supabase.io/new/undefined and only displays a spinner forever. No new organization gets created.
![image](https://user-images.githubusercontent.com/45239725/128064455-22e90612-a053-41f0-a022-bc4f91a65901.png)

**Comment by silentworks** [comment]
Please send an email to beta@supabase.io for this type of issue.

**Comment by inian** [comment, marked as answer]
Since you open an issue as well, follow https://github.com/supabase/supabase/issues/2697 for updates
PS - please dont open both issues and discussions
